migrations/2022-06-21-123144_language-tags/up.sql

```diff
@@ -0,0 +1,195 @@
+insert into language(code, name) values ('ru', 'Русский');
+insert into language(code, name) values ('rw', 'Ikinyarwanda');
+insert into language(code, name) values ('sa', 'संस्कृतम्');
+insert into language(code, name) values ('sc', 'sardu');
+insert into language(code, name) values ('sd', 'सिन्धी');
+insert into language(code, name) values ('se', 'Davvisámegiella');
+insert into language(code, name) values ('sg', 'yângâ tî sängö');
+insert into language(code, name) values ('si', 'සිංහල');
+insert into language(code, name) values ('sk', 'slovenčina');
+insert into language(code, name) values ('sl', 'slovenščina');
+insert into language(code, name) values ('sm', E'gagana fa\'a Samoa');
+insert into language(code, name) values ('sn', 'chiShona');
+insert into language(code, name) values ('so', 'Soomaaliga');
+insert into language(code, name) values ('sq', 'Shqip');
+insert into language(code, name) values ('sr', 'српски језик');
+insert into language(code, name) values ('ss', 'SiSwati');
+insert into language(code, name) values ('st', 'Sesotho');
+insert into language(code, name) values ('su', 'Basa Sunda');
+insert into language(code, name) values ('sv', 'Svenska');
+insert into language(code, name) values ('sw', 'Kiswahili');
+insert into language(code, name) values ('ta', 'தமிழ்');
+insert into language(code, name) values ('te', 'తెలుగు');
+insert into language(code, name) values ('tg', 'тоҷикӣ');
+insert into language(code, name) values ('th', 'ไทย');
+insert into language(code, name) values ('ti', 'ትግርኛ');
+insert into language(code, name) values ('tk', 'Türkmençe');
+insert into language(code, name) values ('tl', 'Wikang Tagalog');
+insert into language(code, name) values ('tn', 'Setswana');
+insert into language(code, name) values ('to', 'faka Tonga');
+insert into language(code, name) values ('tr', 'Türkçe');
+insert into language(code, name) values ('ts', 'Xitsonga');
+insert into language(code, name) values ('tt', 'татар теле');
+insert into language(code, name) values ('tw', 'Twi');
+insert into language(code, name) values ('ty', 'Reo Tahiti');
+insert into language(code, name) values ('ug', 'ئۇيغۇرچە‎');
+insert into language(code, name) values ('uk', 'Українська');
+insert into language(code, name) values ('ur', 'اردو');
+insert into language(code, name) values ('uz', 'Ўзбек');
+insert into language(code, name) values ('ve', 'Tshivenḓa');
+insert into language(code, name) values ('vi', 'Tiếng Việt');
+insert into language(code, name) values ('vo', 'Volapük');
+insert into language(code, name) values ('wa', 'walon');
+insert into language(code, name) values ('wo', 'Wollof');
+insert into language(code, name) values ('xh', 'isiXhosa');
+insert into language(code, name) values ('yi', 'ייִדיש');
+insert into language(code, name) values ('yo', 'Yorùbá');
+insert into language(code, name) values ('za', 'Saɯ cueŋƅ');
+insert into language(code, name) values ('zh', '中文');
+insert into language(code, name) values ('zu', 'isiZulu');
```

Comment:
Is the name column really necessary? After all any front end app is going to have to create the user interface from the code given.

Maybe that's necessary for activitypub?

Reply:
Exactly, it should federate in the same way as Peertube, which looks like this. And as we already have the language names, we can also serve them through the api for use in frontends, making implementation a bit simpler.

```
  "language": {
    "identifier": "en",
    "name": "English"
  },
```

Reply:
Gotcha.